tests/v2/test_waku_noise.nim

```diff
@@ -0,0 +1,82 @@
+{.used.}
+
+import
+  testutils/unittests,
+  ../../waku/v2/protocol/waku_noise/noise,
+  ../test_helpers,
+  std/tables
+
+procSuite "Waku Noise":
+  
+  let rng = rng()
+
+  test "Encrypt -> decrypt public keys":
+
+    let noisePublicKey: NoisePublicKey = genNoisePublicKey(rng[])
+
+    let 
+      cs: ChaChaPolyCipherState = randomChaChaPolyCipherState(rng[])
+      enc_pk: NoisePublicKey = encryptNoisePublicKey(cs, noisePublicKey)
```

Comment:
Prefer using `camelCase` for all identifiers for consistency. See: https://nim-lang.org/docs/nep1.html#introduction-naming-conventions

Reply:
Addressed in https://github.com/status-im/nim-waku/commit/704f3fe40fb4ad63212ec5f6face687b4971c909